Frameworks/CoreFoundation/PlugIn.subproj/CFBundle_Locale.c

```diff
@@ -531,6 +536,10 @@ CF_PRIVATE CFArrayRef _CFBundleCopyUserLanguages() {
     static CFArrayRef _CFBundleUserLanguages = NULL;
     static dispatch_once_t once = 0;
     dispatch_once(&once, ^{
+// WINOBJC: __CFAppleLanguages does not exist on Windows
+#if DEPLOYMENT_TARGET_WINDOWS
+        _CFBundleUserLanguages = EnumerateUserPreferredLanguages();
```

Comment:
Per offline suggestion, I'd prefer we use `CFLocaleGetPreferredUserLanguages` or whatever it's called. That way we don't need to use our WinRT `Enumerate` directly everywhere.

Reply:
minor: should we be transforming this list from `-` to `_` on creation? it could save us the step of converting the `-` to a `_` for every lookup.

Reply:
we could do it inside Enumerate if CF prefers `_` locale names

Reply:
Windows locales are set up to use `-`. If somebody is setting up localization, they may think to use the natural windows syntax in their bundle names. I think it makes sense to support both `-` and `_` in that case.